src/Config/Main.hs

```diff
@@ -36,6 +38,15 @@ import UI.ComposeEditor.Keybindings
 import Types
 import Storage.Notmuch (getDatabasePath)
 
+sendmailPath :: FilePath
+sendmailPath = "/usr/sbin/sendmail"
+
+renderSendMail :: B.ByteString -> IO String
+renderSendMail m =
+        -- -t which extracts recipients from the mail
+        -- TODO: accept a ByteString as stdin
```

Comment:
Yeah, file a ticket for this TODO as well.  Although maybe it can go in the same ticket as the other `readProcess` problems we already discussed.